cpp/basix.h

```diff
@@ -17,6 +17,11 @@ void release_element(int handle);
 std::vector<Eigen::ArrayXXd> tabulate(int handle, int nd,
                                       const Eigen::ArrayXXd& x);
 
+/// Apply mapping
```

Comment:
What map? I could suggest a name but I don't know from the docstring what it does. How many point does it work on? Can the operation it performs be described?

Reply:
I've renamed this to map_push_forward and added a better docstring.